### PR TITLE
Issue 303 - better feedback after saving post

### PR DESCRIPTION
**Pull request by hollycorbett**
Basic functionality after saving post.

Do we want to push this out to a generic directive based off the modal directive?  I've gone back and forth about it a couple of times for the sake of issue 468, but in this case its without.  Can easily change.

Delete and error notification coming soon after based on same approach.  Let me know if you want a different style from this - I was largely intending on doing a c+p job.

[Review on Reviewable](https://reviewable.io/reviews/ushahidi/platform-client/21)

### CODE REVIEW

**Comment by rjmackay** [comment]
erm.. I think you borked the PR? Its pulling in changes to do with feature toggling.. but nothing to do with posts feedback? Wrong branch?

**Comment by hollycorbett** [comment]
Okay.  This time.  Colour me confuzzled.  Travis still hates me, but lets say thats on my list for figuring out later.  Currently doing the same thing as in this one for deletioning posts.

File: app/post/controllers/post-detail-controller.js [hunk 1]
```diff
@@ -140,13 +142,19 @@ function (
...
+    };
+
+    $scope.cancelDelete = function () {
+        // again - feel there must be a more angular-ish way to do this
+        $scope.userRequestedPostDelete = false;
+    };
+
     $scope.deletePost = function () {
-        $translate('notify.post.destroy_confirm').then(function (message) {
-            if (window.confirm(message)) {
```
Comment (rjmackay) [inline app/post/controllers/post-detail-controller.js]: This looses any confirmation before deleting a post??

File: server/www/templates/posts/post-editor.html [hunk 2]
```diff
@@ -297,4 +297,13 @@ <h1 translate translate-values="{ type: activeForm.name, title: post.title }" ng
             </p>
         </div>
     </div>
+
+    <modal visible="userSavedPost">
+        <h1 translate>post.save_dlg_title</h1>
+        <div translate translate-values="{ 'post': post.title }">post.you_have_published</div>
+        <div class="modal-submit">
+            <button type="button" class="button-primary" ng-click="navigateToPost()" translate>post.see_post</button>
```
Comment (rjmackay) [inline server/www/templates/posts/post-editor.html]: If you need a button to change URL, just switch it to a link `<a href="/posts/id" class="button button-primary"></a>

File: server/www/templates/posts/post-editor.html [hunk 3]
```diff
@@ -297,4 +297,13 @@ <h1 translate translate-values="{ type: activeForm.name, title: post.title }" ng
             </p>
         </div>
     </div>
+
+    <modal visible="userSavedPost">
```
Comment (rjmackay) [inline server/www/templates/posts/post-editor.html]: .. this should really be a confirmation message as per the pattern: http://github.ushahidi.org/platform-pattern-library/assets/html/4_blocks/#pl-pattern-confirmation

.. but this is better than alerts for launch

**Comment by rjmackay** [comment]
Looks usable in the short term.. long term we need something more easily reusable (ie. where you can just call a function and get a promise that'll fire when confirmed or cancelled)
I think that probably looks like a single confirmation modal which is triggered from the notify service or something similar.

**Comment by rjmackay** [comment]
@willdoran can you extend this to give us quota messages in time for launch?

**Comment by willdoran** [comment]
I hope so!

File: app/post/controllers/post-detail-controller.js (hunk 1 above)
Comment (hollycorbett) [inline app/post/controllers/post-detail-controller.js]: Nope, cause this function now only gets called when the deletion is confirmed in the modal.

**Comment by rjmackay** [comment]
This already landed along with Will's changes .. sorry should have closed this earlier

**Pull request closed without merging**